docs/source/guides/contributor/chapters/plugins.rst

```diff
@@ -1,11 +1,11 @@
 Writing an Avocado plugin
--------------------------
+*************************
```

Comment:
Where do these come from? I mean, my emacs RST mode gives me different section decorators automatically, and this seems to suggest different order of section levels: https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html#headings

Reply:
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

I'm trying to use chapters starting with "****" so we could have more sub-sections.

Reply:
OK, doesn't matter much to me.  But, for the record, it says:

```
# with overline, for parts
* with overline, for chapters
```

That is, chapters it should be:

```
*************************
Writing an Avocado plugin
*************************
```

If using `*`.

Reply:
Not sure if you are talking about the top line, but it is optional:

> 
Section headers (ref) are created by underlining (and optionally overlining) ...

Reply:
> Not sure if you are talking about the top line, but it is optional:
> 
> > Section headers (ref) are created by underlining (and optionally overlining) ...

Yes, overlines for *sections* seem to optional... but not for parts or chapters it seems.  See that both `#` and `*` require them, as per your linked reference:

```
# with overline, for parts
* with overline, for chapters
=, for sections
-, for subsections
^, for subsubsections
", for paragraphs
```

But this is the ultimate nitpicking, and seems to be only style issues, so don't mind it.

Reply:
It is better to be fully compatible though and such corrections don't take much time. There are also unforgiving rst parsers out there. But from the current changes I see, the overline is already there too.